src/whisper_at_train/run_combined_training.sh

```
@@ -7,11 +7,33 @@
+# Get the script directory for relative path resolution
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "${SCRIPT_DIR}/../.." && pwd)"
+
+# Change to script directory to ensure relative paths work correctly
+cd "$SCRIPT_DIR"
+
+# Environment setup - check if running in Poetry environment
+if command -v poetry &> /dev/null && poetry env info --path &> /dev/null; then
+    echo "Using Poetry environment"
+    # Use source instead of poetry shell for non-interactive environments
+    source "$(poetry env info --path)/bin/activate"
```

Shellcheck (0.10.0)

[info] Not following: ./bin/activate was not specified as input (see shellcheck -x).

(SC1091)